testbed/datareceivers/mockawsxraydatareceiver.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datareceivers
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configtls"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/testbed/testbed"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/mockdatareceivers/mockawsxrayreceiver"
+)
+
+// MockAwsXrayDataReceiver implements AwsXray format receiver.
+type MockAwsXrayDataReceiver struct {
+	testbed.DataReceiverBase
+	receiver component.TraceReceiver
+}
+
+// NewMockAwsXrayDataReceiver creates a new  MockDataReceiver
+func NewMockAwsXrayDataReceiver(port int) *MockAwsXrayDataReceiver {
+	return &MockAwsXrayDataReceiver{DataReceiverBase: testbed.DataReceiverBase{Port: port}}
+}
+
+//Start listening on the specified port
+func (ar *MockAwsXrayDataReceiver) Start(tc consumer.TracesConsumer, _ consumer.MetricsConsumer, _ consumer.LogsConsumer) error {
+	var err error
+	os.Setenv("AWS_ACCESS_KEY_ID", "AWS_ACCESS_KEY_ID")
+	os.Setenv("AWS_SECRET_ACCESS_KEY", "AWS_SECRET_ACCESS_KEY")
```

Comment:
```suggestion
        // Set unrealistic values for keys so AWS SDK does not produce auth errors even though
        // this is a mock.
	os.Setenv("AWS_ACCESS_KEY_ID", "placeholder")
	os.Setenv("AWS_SECRET_ACCESS_KEY", "placeholder")
```

I think the value of placeholder makes more clear this has no meaning